scopes/workspace/install/install.main.runtime.ts

```diff
@@ -174,22 +163,79 @@ export class InstallMain {
       packageImportMethod: this.dependencyResolver.config.packageImportMethod,
       rootComponents: hasRootComponents,
     };
-    await installer.install(
-      this.workspace.path,
-      mergedRootPolicy,
-      compDirMap,
-      { installTeambitBit: false },
-      pmInstallOptions
-    );
+    // TODO: pass get install options
+    const installer = this.dependencyResolver.getInstaller({});
```

Comment:
@zkochan do we need to implement this todo before the merge?

Reply:
This TODO was there before my changes. I just moved the code.